sphinxarg/ext.py

```diff
@@ -142,10 +142,9 @@ def print_action_groups(data, nested_content, markdown_help=False, settings=None
                     '"==SUPPRESS=="',
                     '==SUPPRESS==',
                 ]:
-                    if entry['default'] == '':
-                        arg.append('Default: ""')
-                    else:
-                        arg.append(f"Default: {entry['default']}")
+                    # Put the default value in a literal block - but escape backticks already in the string
+                    default_str = str(entry['default']).replace('`', r'\`')
```

Comment:
Small problem: the 
```
\`
```
is displayed in the webbrowser literally. I haven't been able to property escape it in Sphinx to display it properly in a literal block.

Reply:
Hm, actually in plain Sphinx this works fine. The following is rendered as expected:
```
Demo code: ``hi``

Demo code: `````

Demo code: ``\```

Demo code: ```hi```

Demo code: ````hi````

Demo code: ``meta: ``hi````
```

![image](https://github.com/sphinx-doc/sphinx-argparse/assets/12717020/a1ef18ba-3968-4d85-b9d4-bb2e1199354d)